docs/modules/configuration/pages/jet-configuration.adoc

```diff
@@ -297,6 +297,18 @@ The most important properties are listed here:
 Each job has job-specific configuration options. These are covered
 in detail in xref:pipelines:configuring-jobs.adoc[].
 
+=== Isolated Jobs
+
+To activate isolated jobs, your license key must include `Advanced Compute`.
+
+To isolate computation from storage, you can define the location for Jet jobs. This means that you can manage your workload without worrying that the Jet processing jobs starve resources from your storage components.
+
+NOTE: Your storage components still need to serve the data and this has some impact on their resources. Before using isolated jobs, ensure that the processing element of the job is substanitally more resource-intensive than the data retrieval element. 
+
+Use the `JobBuilder` API to define the default behaviour for your isolated computing. For further information on `JobBuilder`, refer to the link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/jet/JetService.JobBuilder.html[API Reference, window=_blank].
+
+You can override this configuration when you submit your job from the Hazelcast client. For more information on submitting an isolated job, see xref:pipelines:submitting-jobs.adoc#isolated-jobs[Submitting Jobs].
```

Comment:
Changing the selector requires re-submitting the job, so I don't think this qualifies as "overriding"

Reply:
Reworded